library/core/src/ffi/primitives.rs

```diff
@@ -0,0 +1,174 @@
+//! Defines primitive types that match C's type definitions for FFI compatibility.
+//!
+//! This module is intentionally standalone to facilitate parsing when retrieving
+//! core C types.
+
+macro_rules! type_alias {
+    {
+      $Docfile:tt, $Alias:ident = $Real:ty;
+      $( $Cfg:tt )*
+    } => {
+        #[doc = include_str!($Docfile)]
+        $( $Cfg )*
+        #[stable(feature = "core_ffi_c", since = "1.64.0")]
```

Comment:
Can we mark this as something like stable_internal for private modue?
Or just #[stable(feature = "", since = "")]

Reply:
done, I found way to deal with it